cmd/mdatagen/internal/samplereceiver/internal/metadatatest/generated_telemetrytest.go

```diff
@@ -19,15 +21,21 @@ import (
 )
 
 type Telemetry struct {
+	Spanrecorder *tracetest.SpanRecorder
+
 	reader        *sdkmetric.ManualReader
 	meterProvider *sdkmetric.MeterProvider
+	traceProvider *sdktrace.TracerProvider
```

Comment:
Need to shutdown this as well.

Reply:
updated in https://github.com/open-telemetry/opentelemetry-collector/pull/12106/commits/365a0680018d0bfba509ad3b7dcd768d06bd123f